sklearn/utils/estimator_checks.py

```diff
@@ -960,6 +960,8 @@ def check_estimators_pickle(name, Estimator):
 
     for method in result:
         unpickled_result = getattr(unpickled_estimator, method)(X)
+        result[method] = np.asarray(result[method]).flatten()
+        unpickled_result = np.asarray(unpickled_result).flatten()
```

Comment:
Could you explain why you have to add these lines to this common test? This is what is causing the travis failure, and it looks like an actual bug if you need to modify the common test this way.  What is the intended behavior?